public/data/Lavakush Biyani.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Lavakush Biyani",
+  "type": "personal",
+  "bio": "I am Lavakush Biyani, currently pursuing my B.tech. I am a tech enthusiast & an open-source contributor. I am always open to collaborating on projects and innovative/disruptive ideas",
+  "avatar": "https://github.com/lavakush07.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/lavakush07",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/lavkushbiyani1",
+      "icon": "twitter"
+    },
+    {
+     "name":"Follow me on Linkedin",
+      "url":"https://www.linkedin.com/in/lavkush-biyani-07/",
+      "icon": "Linkedin"
```

Comment:
```suggestion
      "icon": "linkedin"
```

Reply:
@Panquesito7 it doesn't matter if this is uppercase or lowercase, we have a function in the code to deal with this :)

Reply:
It's still there, it's in the ProfilePage.js component